app/views/bookings/new.html.erb

```diff
@@ -1,6 +1,18 @@
-<%= simple_form_for @booking do |f| %>
-<%= f.association :user %>
-<%= f.association :space %>
-<%= f.input :day %>
-<%= f.submit "Confirm booking", class: "btn btn-primary" %>
-<% end %>
\ No newline at end of file
+<div class="container">
+  <div class="row">
+    <div class="col-12 col-md-6 offset-md-3">
+
+    <%= simple_form_for @booking do |f| %>
+        <div class="row justify-content-center"></div>
```

Comment:
Same as previous comment regarding form styling. Please work with @jselvaraj95 to ensure all form styles are uniform.